bears/general/FilenameBear.py

```diff
@@ -98,14 +101,19 @@ def run(self, filename, file,
                 'Filename does not use the suffix {!r}.'.format(
                     filename_suffix))
 
+        if len(new_name+extension) > max_filename_length:
```

Comment:
Hm okay you are analyzing the new name generated. We should be careful not to confuse the user here, because he thinks "huh, but my file isn't called like that / the length of the name doesn't match up :O".

Reply:
What do you think about adding whole name of file (I mean new_name + extension) in error message? :)

Reply:
still not sure...

Imo I would use the "old" filename for this check, I think even displaying `new_name + extension` would confuse the user. For now I would say we just use the old name, if users start to complain about this setting, we think about it again ;)

Reply:
I'm still not sure. You should imagine such scenario : 
If we'd check filename on the start, then we aren't catch too long filename after adding required suffix or prefix (this is possible).
So in my opinion checking this on new_name is better solution.